Evolve.Migrations.Helper/Program.cs

```diff
@@ -44,8 +50,8 @@ private static bool ValidateArguments(IReadOnlyList<string> args)
         {
             if (args.Count == 0 || args[0] == "--help")
             {
-                Console.WriteLine("Evolve Migrations Helper\r\n");
-                Console.WriteLine($"Usage: {Assembly.GetExecutingAssembly().GetName().Name} [command] [options]\r\n");
+                PrintHeaderMessage();
+                Console.WriteLine($"\r\nUsage: {Assembly.GetExecutingAssembly().GetName().Name} [command] [options]\r\n");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/odair-pedro/evolve-migrations-helper/pullRequest?prid=5618245)